src/Modules/Windows/Microsoft.PowerShell.Utility/Microsoft.PowerShell.Utility.psd1

```diff
@@ -29,4 +29,14 @@ FunctionsToExport = @()
 AliasesToExport = @('fhx')
 NestedModules = @("Microsoft.PowerShell.Commands.Utility.dll")
 HelpInfoURI = 'https://go.microsoft.com/fwlink/?linkid=855960'
+PrivateData = @{
+  PSData = @{
+    ExperimentalFeatures = @(
+      @{
+        Name = 'Microsoft.PowerShell.Utility.PSDebugRunspaceWithBreakpoints'
+        Description = "Enables the New-PSBreakpoint cmdlet and the -Breakpoint parameter on Debug-Runspace to set breakpoints in another Runspace upfront."
```

Comment:
@daxian-dbw: I'm glad it is listed as experimental, because there are some challenges with the design. See: https://github.com/PowerShell/PowerShell/pull/8923#issuecomment-502471222.

Reply:
FYI, the `New-PSBreakpoint` cmdlet has been removed in a PR that should be submitted today. I've also removed the experimental feature flag, because I think it is not needed anymore. We can work out details in that PR, and I'll happily add back the experimental feature flag (fully, including what was missed the first time) if necessary. This PR can probably be closed once I have that PR open, so I'll comment back here soon.

Reply:
> I've also removed the experimental feature flag, because I think it is not needed anymore.

The experimental feature flag is still needed. That will be my first piece of feedback.

Reply:
But you haven't even seen the code... 😛